index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>SURVEY FORM</title>
+    <title>SURVEY FORMS</title>
 </head>
 <body>
     <h1 id="title">freecodecamp Survey Form</h1>
```

Comment:
Hello, I suggest you use `<header>` tag, it is a semantic html tag. It improves the productivity of your code.

Reply:
> Hello, I suggest you use `<header>` tag, it is a semantic html tag. It improves the productivity of your code.

- [ ] Good comment :+1: , so what do you think about adding some link that support your argument? :+1:. It will help the person you are reviewing learn more about this kind of error, so they won't redo them :call_me_hand:

Reply:
As the previous reviewer requested, please add a link to help that person to know more about the related topics. 📝🔗

Reply:
As the previous reviewers have said, please add a link to an article that at least talks about the header tag. Consider checking mdn docs for one.